packages/client-sdk-nodejs/src/cache-client.ts

```diff
@@ -33,6 +34,7 @@ export class CacheClient extends AbstractCacheClient implements ICacheClient {
   private readonly logger: MomentoLogger;
   private readonly notYetAbstractedControlClient: CacheControlClient;
   private readonly configuration: Configuration;
+  private semaphore: Semaphore;
```

Comment:
Let's give this a meaningful name like "dataRequestConcurrencySemaphore", or whatever you think appropriate, since this is for limiting concurrency of data requests.